Mehdi_test/date_f.nit

```diff
@@ -0,0 +1,104 @@
+#Module date_f has the basic date fuction implementation
```

Comment:
Licence for the file please, take any file from the repo and copy it inside this one